src/pasteup/numberSVGs.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+
+export default ({ index }) => {
+    const numbers = {
+        0: (
+            <svg width="34" height="40">
+                <path d="M17 40C7.8 40 0 33.2 0 20.2 0 6.6 8.1 0 17.3 0 26.4 0 34 6.5 34 19.9S26.3 40 17 40zm.1-1c4 0 5.2-6.1 5.2-18.7C22.3 7 21 1.1 17.1 1.1c-4 0-5.4 6-5.4 18.7 0 13 1.4 19.1 5.4 19.2z" />
+            </svg>
+        ),
+        1: (
+            <svg width="21" height="40">
+                <path
+                    d="M0 40h21v-1l-5.4-1.6V0h-4.3L.1 5.4v1.3l5.5-1.4v32.1L0 39.1z"
+                    fillRule="evenodd"
```

Comment:
appreciate this was a copy and paste, but i don't think attribute will actually do anything here?